MITRE ATT&CK for Azure Sentinel.ipynb

```diff
@@ -0,0 +1,2063 @@
+{
```

Comment:
<p>This is slow, maybe have a progress bar or print statement just saying it working.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>added print statements</p>